website/docs/d/cloudaccount_oci.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "dome9"
+page_title: "Check Point CloudGuard Dome9: dome9_cloudaccount_oci"
+sidebar_current: "docs-datasource-cloudaccount-oci"
+description: |-
+  Get information about oci cloud account onboarded to Dome9.
```

Comment:
in case the "oci" is short for a more meaningful name the description or even the name of the page can be better with the long name because the user may identify it better

Reply:
OCI is like AWS